aplus/celery.py

```diff
@@ -16,6 +21,10 @@ def setup_periodic_tasks(sender, **kwargs):
     if hasattr(settings, 'SIS_ENROLL_SCHEDULE'):
         sender.add_periodic_task(settings.SIS_ENROLL_SCHEDULE, enroll.s(), name='enroll')
 
+@app.on_after_configure.connect
+def setup_periodic_tasks(sender, **kwargs):
+    sender.add_periodic_task(settings.SUBMISSION_EXPIRY_TIMEOUT, retry_submissions.s(), name='retry_submissions')
```

Comment:
I could do one more modification to this PR: possibility to disable this feature from settings, e.g. by setting SUBMISSION_EXPIRY_TIMEOUT to None, so that it would be possible to turn this off, if some unexpected problems show up after deployment.

Reply:
Good idea!